docs/README.md

```diff
@@ -77,6 +77,17 @@ const LocalValue = () =>{
   );
 }
 ```
+### Parameters
+1. `inputFuction` (_any_) : Input Function of the state.
+2. `delay` (_String_) : delay for the debounce.
+
+### Return value
+
+`[Callback, setCallback]`
```

Comment:
There is some misunderstanding here. What this function returns is another function. Please refer the code of  useDebounce hook

Reply:
can you please explain me this part again

Reply:
Refer [this](https://github.com/aromalanil/useCustomHooks/blob/master/hooks/useDebounce.js#L27)

This hook returns a modified function, which is forced to wait a certain amount of time before running again.